etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/database/PostgresDaoTest.groovy

```diff
@@ -148,17 +148,18 @@ class PostgresDaoTest extends Specification {
     def "fetchMetadata returns partnermetadata when rows exist"() {
         given:
         def messageId = "12345"
-        def receiver = "DogCow"
+        def sender = "DogCow"
         Timestamp timestampForMock = Timestamp.from(Instant.parse("2024-01-03T15:45:33.30Z"))
         Instant timeReceived = timestampForMock.toInstant()
-        def hash = receiver.hashCode().toString()
-        def expected = new PartnerMetadata(messageId, receiver, timeReceived, hash)
+        def hash = sender.hashCode().toString()
```

Comment:
Should the changes to the DB implementation in this PR be in the other PR given is not directly related to the marshaling of the response?

Reply:
The above changes are fixing a bug I had from previous PRs, because of how I was retrieving results, one column from the db was never being populated.  So I updated the above test after fixing the results retrieval as the test started to fail.